test/draft.test.ts

```diff
@@ -38,7 +33,7 @@ suite("draft tests", () => {
         suite("If draft is not on the path", () => {
 
             test("...and configuration is not present, then checkPresent reports an error", async () => {
-                let errors: string[] = [];
+                const errors: string[] = [];
```

Comment:
Ugh, this is particularly misleading because the `errors` collection gets things pushed onto it _via being passed as an argument_, so the mutation is very non-obvious.  (I guess the fix is to make the mutation more obvious somehow...)

Reply:
"fixed" all of these in so far as i left them as `let` with a disable condition for the tslint rule.